test/extensions/filters/http/lua/lua_integration_test.cc

```diff
@@ -908,7 +908,7 @@ name: basic_lua_routes
     route:
       cluster: lua_cluster
     typed_per_filter_config:
-      envoy.filters.http.lua:
+      lua:
```

Comment:
`envoy.filters.http.lua` will still be an valid key as long as it is also specified in [filter config name](https://github.com/envoyproxy/envoy/blob/65b1f6039236e466c196e3cfbe082567c569b640/envoy/http/filter_factory.h#L33), right?

In other words, there is no favoring of  `lua` over `envoy.filters.http.lua` from naming perspective?